docs/contributors/getting-started.md

```diff
@@ -0,0 +1,102 @@
+# Getting Started
+
+Before you can begin contributing to the Blocks plugin there are several steps and tools required to setup your local development environment.
+
+## Cloning the Git Repository
+
+Before you can start modifying files you'll want to clone this repository locally, either via the command line, or using a Git client such as [GitHub Desktop](https://desktop.github.com/).
+
+To do so from the command line, ensure you have [`git`](https://git-scm.com) installed on your machine, and run the clone command:
+
+```
+git clone https://github.com/woocommerce/woocommerce-gutenberg-products-block.git
+```
+
+## Installing dependencies
+
+To install dependencies, you will need the following tools installed on your machine:
+
+-   [`npm` and `node.js`](https://nodejs.org)
+-   [`composer`](https://getcomposer.org)
+
+See [`package.json` `engines`](package.json) for details of required versions.
+
+Once you have `node` and `composer` setup, install the dependencies from the command line:
+
+-   Change directory to your repo folder, e.g. `cd woocommerce-gutenberg-products-block`.
+-   Install javascript and php dependencies - `npm install && composer install`.
+
+## Building the plugin files
+
+NPM is used to trigger builds. Building is required for the plugin to functional.
+
+-   Run `$ npm run build` to build all assets for production.
```

Comment:
Several of our doc files are a bit inconsistent when using the `$` in front of commands: in this file, for example, line 27 doesn't have `$` but this line does. I don't have any preference, but I wonder if not being consistent might be confusing for contributors?